ps1/flattennpmmodules.ps1

```diff
@@ -14,21 +14,16 @@ If (-Not (Test-path $source)) {
     return "Source directory cannot be found"
 }
 
-$flattenbin = (get-item $PSScriptRoot).parent.FullName + '\node_modules\flatten-packages\bin\flatten'
-If (!(Test-Path $flattenbin)) {
-    $flattenbin = (get-item $PSScriptRoot).parent.FullName + '\..\flatten-packages\bin\flatten'
-}
-
 function flatMe([string]$path){
     if (!$path.EndsWith('\')) {
         $path = $path + '\'
     }
-    
+
     Write-Host "Getting subdirectories of: " $path
     Get-ChildItem $path -Directory | ForEach-Object {
         if($_.Name -eq 'node_modules'){
             Write-Host "Flattening: " $path
-            node $flattenbin $path
+            npm dedupe
```

Comment:
Wait a second... this just runs `npm dedupe` in the current `cwd`. That's not the goal though.